headers/types/ground_mode/ground_mode.h

```diff
@@ -0,0 +1,19 @@
+// Types used during the ground part (overlay 11)
+
+#ifndef HEADERS_TYPES_GROUND_MODE_H_
+#define HEADERS_TYPES_GROUND_MODE_H_
+
+#include "partner_following.h";
+
+// A global structure holding various pointer to important structure
+struct main_ground_data {
+    undefined* script; //0x0: pointer to script structure
```

Comment:
Some [formatting issues](https://github.com/UsernameFodder/pmdsky-debug/runs/5336315279?check_suite_focus=true) with the comments. Need a space after the `//`. You can also [download the formatted file](https://github.com/UsernameFodder/pmdsky-debug/actions/runs/1898774927#artifacts) directly.